redis/client.py

```diff
@@ -402,7 +413,7 @@ def parse_georadius_generic(response, **options):
 
     # zip all output results with each casting functino to get
     # the properly native Python value.
-    f = [nativestr]
+    f = [text_or_binary]
```

Comment:
This could simply be a noop call. Something like: `f = [lambda x: x]`

Reply:
Quite right.